qiskit_experiments/library/calibration/analysis/fine_amplitude_analysis.py

```diff
@@ -28,44 +28,51 @@
 class FineAmplitudeAnalysis(CurveAnalysis):
     r"""Fine amplitude analysis class based on a fit to a cosine function.
 
-    Analyse a fine amplitude calibration experiment by fitting the data to a cosine function.
-    The user must also specify the intended rotation angle per gate, here labeled,
-    :math:`{\rm apg}`. The parameter of interest in the
-    fit is the deviation from the intended rotation angle per gate labeled :math:`{\rm d}\theta`.
-    The fit function is
-
-    .. math::
-        y = {\rm amp}/2\cos\left(x[{\rm d}\theta + {\rm apg} ]+{\rm phase\_offset}\right)+baseline
-
-    Fit Parameters
-        - :math:`amp`: Amplitude of the oscillation.
-        - :math:`baseline`: Base line.
-        - :math:`{\rm d}\theta`: The angle offset in the gate that we wish to measure.
-
-    Initial Guesses
-        - :math:`amp`: The maximum y value less the minimum y value.
-        - :math:`baseline`: The average of the data.
-        - :math:`{\rm d}\theta`: Zero.
-
-    Bounds
-        - :math:`amp`: [-2, 2] scaled to the maximum signal value.
-        - :math:`baseline`: [-1, 1] scaled to the maximum signal value.
-        - :math:`{\rm d}\theta`: [-pi, pi].
-
-    Fixed-value parameters:
-        - :math:`{\rm apg}` The angle per gate is set by the user, for example pi for a pi-pulse.
-        - :math:`{\rm phase\_offset}` The phase offset in the cosine oscillation, for example,
+    # section: fit_model
+
+        Analyse a fine amplitude calibration experiment by fitting the data to a cosine function.
+        The user must also specify the intended rotation angle per gate, here labeled,
+        :math:`{\rm apg}`. The parameter of interest in the
+        fit is the deviation from the intended rotation angle per gate labeled
+        :math:`{\rm d}\theta`. The fit function is
+
+        .. math::
+            y = \frac{{\rm amp}}{2}\cos\left(x[{\rm d}\theta + {\rm apg} ] \
+            +{\rm phase\_offset}\right)+{\rm base}
+
+    # section: fit_parameters
+        defpar \rm amp:
+            desc: Amplitude of the oscillation.
+            init_guess: The maximum y value less the minimum y value.
+            bounds: [-2, 2] scaled to the maximum signal value.
+
+        defpar \rm base:
+            desc: Base line.
+            init_guess: The average of the data.
+            bounds: [-1, 1] scaled to the maximum signal value.
+
+        defpar d\theta:
+            desc: The angle offset in the gate that we wish to measure.
+            init_guess: Zero.

```

Comment:
Is this correct? I think it scans over multiple guesses.

Reply:
Good catch. Thanks: e4f423f